source/administration/changelog.md

```diff
@@ -2,13 +2,28 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.5.0  
-
-Release date: 2016-11-16
+## Release v3.5.1  
+
+### Notes on Patch Release
+
+ - **v3.5.1, released 2016-11-23**
+   - Security update to preventing cross-site scripting and remote code execution, thanks to Harrison Healey for [reporting responsibly](http://www.mattermost.org/responsible-disclosure-policy/).
+   - Fixed an issue where usernames would sometimes not appear beside posts and the reply arrow would throw an error. 
+   - The channel purpose is no longer cut off in the user interface of the **More...** channel menu. 
+   - Fix scroll issue where the center channel didn't always scroll to the bottom when switching channels.
+   - Fix wildcard searching using an asterisk.
```

Comment:
Maybe change this to
```
- Fixed a server error that occurred when searching for users using an asterisk
```
just to make it clear that searching for posts wasn't affected.